### ISSUE TITLE
Accessibility screen-reader audit and past accessibility tickets (Streamlined Waiver, VSA Debt)

### ISSUE THREAD

**Issue by charlottecesana** [department-of-veterans-affairs/va.gov-team#66288]
## Issue Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

**UXR performed a screen-reader audit of the Streamlined Waiver on 9/21/23**

**Overall findings:** 
- Voiceover repeats certain headers, selections, and questions more than once and sometimes more than twice. This will frustrate Veterans who use screen readers, adding significant time burden and cognitive demand to filling out an already demanding form. 
- The tabbing experience seems inconsistent. Tabbing into the form, it reads different page headers or elements. This should be consistent across pages and sections so the Veteran knows what to expect. 
- In several sections, it reads the bulleted lists with the square bullet point as "square bullet" before every line of text. 
- Efforts should be made to simplify the VoiceOver of the eFSR experience to help the Veteran using a screen reader maximize efficiency, build proficiency, enable auditory scannability, and allow flexibility. 

This issue details the issues found in that audit and accessibility tech debt leftover from other tickets. 
Resolving these will take multiple sprints, and doing so will significantly improve the screen reader experience of the Streamlined Waiver. 

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

Accessibility issues found in audit on 9/21

- [ ] Persistent and impactful problem spanning multiple pages: Voice Over reads certain headers, questions, and fields twice or more (elongates the form experience by repeating text, redundant) -  example: "Step 1 of 7: Veteran information"

- [ ] [Intro Page] Reads "Black square" > "Work history for the past 2 years." 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBa3pWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ea7f6636e1bbe80fd15b7e0e55899cecce2a34a8/Screenshot%202023-09-20%20at%208.18.33%20AM.png)
>>

- [ ] [Step 1 of 7: Veteran information] Reads upper navigation (breadcrumbs) ">" as 'quotation mark' between each link
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBazdWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--3c6425d6bac433e46b42ea17e3558053b7df0984/Screenshot%202023-09-20%20at%208.24.54%20AM.png)

>>
- [ ] [Step 1 of 7: Veteran Information: What debt do you need help with?] - It does not introduce the checkbox element early enough. It could be confusing. Sometimes reads text twice (is this as intended?)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBay9WIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--dffb461a3df7c530815ae6173ca408f4d19147e7/Screenshot%202023-09-20%20at%208.32.42%20AM.png)

>>
- [ ] [Step 1 of 7: Veteran Information: Contact Information] - Reads the phone number as "7 billion…" and not individual digits
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbFBWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4032eb417a7ea64002fe05bc78f03a433dcf44ea/Screenshot%202023-09-20%20at%201.24.02%20PM.png)

>>
- [ ] [Step 1 of 7: Veteran Information: Your Dependents]
- [ ] How many dependents? Reads "Black square"
- [ ] Ordering of what VoiceOver reads seems confusing: "First dependent's age> Enter each dependent's age separately > Enter text field

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbHpWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--1ab71fcd14720127bbc9ad342ec85d5e08df0d6e/Screenshot%202023-09-20%20at%202.03.01%20PM.png)
>>

- [ ] [Step 2 of 7: Household Income: Your Work History]
- [ ] Radio button selection can be improved, reads: Radio button 2 of 2 before reading the question … 
- [ ] [Type of work dropdown] can be improved. It does not say the number of options or state when the Veteran reaches the end of the list (may leave Veteran questioning). It reads the first option as empty and does not announce more options below (may leave Veteran confused)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbDNWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--44886bc3a3a0a629d328d304a6cc04320822fa0a/Screenshot%202023-09-21%20at%209.57.28%20AM.png)
>>

- [ ] [Your job at ___ ] - reads all headers twice (Step 2 of 4 … , Your job at ____, Date you started work at this job?)
- [ ] Error message seems out of order (2)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbXZWIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4e205c8c5ad11d81be3532f4d762496b8f649093/Screenshot%202023-09-21%20at%2012.07.15%20PM.png)

>>>
Accessibility issues found previously: 
- [ ] #63674 (Streamlined Waiver)
- [ ] #63675 (Streamlined Waiver)
- [ ] #63680 (Streamlined Waiver)
- [ ] #28349 (VSA Debt)
- [ ] #28343 (VSA Debt)
- [ ] #17894 (VSA Debt)
- [ ] #22733 (VSA Debt)
- [ ] #28937 (VSA Debt)

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
- [ ] Accessibility issues will be resolved 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by HeatherRienks** [comment]
This important work requires FE help and we will not be able to take this work on in Sprint 109 (9/27-10/10) due to competing priorities. We will need to revisit in Backlog Refinement.

**Comment by HeatherRienks** [comment]
Per Backlog Refinement on 10/4, let's create a new Implementation Epic after the AT study is completed.

**Comment by HeatherRienks** [comment]
We will wait until after AT study is completed (around 10/31/23) to add to new epic

**Comment by HeatherRienks** [comment]
Per refinement on 11/15/23, the team plans to review feedback from post- AT study and create new individual tickets, which we will add to a new Epic (#TBD)
